.ci.yaml

```diff
@@ -199,6 +199,8 @@ platform_properties:
       device_type: none
   windows_arm64:
     properties:
+      # The arch can be removed after https://github.com/flutter/flutter/issues/135722.
+      arch: arm64
```

Comment:
Should this be `arm` to match LUCI? See: https://github.com/luci/recipes-py/blob/0a42b2afe8ec90efadfd089f74459ae323c7ab30/recipe_modules/platform/api.py#L21

Reply:
Good catch! Updated.